### PR TITLE
Add support for self-signed SSL certificates

### LINKED ISSUE #26: New certificate chain causing issues
Since today, the new KvK API certificate chain causes issues - a cURL error is returned:

`cURL error 60: Unrecognized Object Identifier. (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)`

I assume this happens for everyone. Message from KvK API Team:

Dear valued costumer,

On September 28, KVK (Chamber of Commerce) will switch to a private root certificate for connecting
to the KVK API Search and KVK API Profile. This implies that all (!) customers need to take immediate
action. An API-key will not be sufficient to provide a connection to the API’s. You need to adapt your
software. This can be achieved by trusting the trust chain in your software.
Instructions can be found https://developers.kvk.nl/guides.

### PR DESCRIPTION
**Pull request by LauLaman**
- Add support for self sigend certificates to KvkClientFactory
- Add dutch government root certificate
- Update tests

fixes #26

### CODE REVIEW

File: src/KvkClientFactory.php [hunk 1]
```diff
@@ -20,23 +19,38 @@
...
         $stack = HandlerStack::create();
         $stack->unshift(Middleware::mapRequest(function (RequestInterface $request) use ($userKey) {
             return $request->withUri(Uri::withQueryValue($request->getUri(), 'user_key', $userKey));
         }));
 
+        if ($rootCertificate === null) {
+            trigger_error('kvk-api: Not using a root certificate is deprecated and will be required in version 1.0. Please configure a root certificate.', E_USER_DEPRECATED);
+        }
+
+        $client = new \GuzzleHttp\Client([
```
Comment (thijsBreker) [inline src/KvkClientFactory.php:47]: Can we `use` this instead?

**Comment by thijsBreker** [comment]
I'm missing a test for the deprecated use-case. Other than that it looks good to me :+1:

File: src/KvkClientFactory.php (hunk 1 above)
Comment (LauLaman) [reply src/KvkClientFactory.php]: Done!

**Comment by LauLaman** [comment]
@thijsBreker this is something that is very hard to test. `E_USER_DEPRECATED` is written directly to the `stdout` so not easily testable. Unless you know a way to test this?

**Comment by LauLaman** [comment]
Can we merge this? A Tests is in place to make sure backward compatibility is still working. The only thing that's super hard to test is the stdout error message.

**Review by thijsBreker — APPROVED** [review]
Yeah, let's merge this :+1:

**Comment by LauLaman** [comment]
@thijsBreker Looks like the automatic tests are stuck..

**Comment by LauLaman** [comment]
@thijsBreker 🛎

**Comment by MarijnKoesen** [comment]
Thanks @LauLaman ! We're looking into the scrutinizer issues, we're hoping to get it merged tomorrow :)

**Comment by Duncank** [comment]
@MarijnKoesen Thanks for merging! Will you create a new release as well so we can update this plugin?

**Comment by LauLaman** [comment]
@Duncank the new release is [v0.1.3](https://github.com/Werkspot/kvk-api/releases/tag/v0.1.3)

**Pull request merged**